src/java/org/jivesoftware/openfire/plugin/session/RemoteClientSession.java

```diff
@@ -40,13 +40,21 @@
  *
  * @author Gaston Dombiak
  */
+
+/*
+* Changed for OF-1868 
+* Openfire Repo has to be updated first, before this fix could be compiled, because 
+* org.jivesoftware.openfire.session.ClientSessionInfo has changed too this class is depending on. 
+*/
+
 public class RemoteClientSession extends RemoteSession implements ClientSession {
 
     private long initialized = -1;
 
-    private boolean messageCarbonsEnabled;
-
-    private boolean hasRequestedBlocklist;
+/*
+*Changed for OF-1868 
+*/   
+    //private boolean hasRequestedBlocklist;
```

Comment:
Don't comment out code - remove it. We have git to record historical stuff.

Reply:
ok next time ;)